broker-daemon/block-order-worker/index.js

```diff
@@ -109,14 +109,37 @@ class BlockOrderWorker extends EventEmitter {
       this.relayer.paymentChannelNetworkService.getAddress({symbol: blockOrder.inboundSymbol})
     ])
 
-    const outboundBalanceIsSufficient = await outboundEngine.isBalanceSufficient(outboundAddress, Big(blockOrder.outboundAmount).plus(activeOutboundAmount))
+    let counterAmount
+    let outboundAmount
+    let inboundAmount
+    if (blockOrder.isMarketOrder) {
+      const { orders, depth } = await orderbook.getBestOrders({ side: blockOrder.inverseSide, depth: Big(blockOrder.baseAmount).toString() })
```

Comment:
why not just use `blockOrder.baseAmount` for depth (instead of doing `Big(...).toString()`?